src/components/Nav/Nav.js

```diff
@@ -1,7 +1,57 @@
 import React, { Component } from 'react';
+import NavMain from './NavMain/NavMain';
+import NavCategory from './NavCategory/NavCategory';
+import SerchModal from './SerchModal/SerchModal';
 
 export default class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isLocationButton: false,
+      isSerch: false,
+      categotyList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('data/navListData.json')
```

Comment:
```suggestion
    fetch('/data/navListData.json')
```
절대경로앞에는 `/`가 반드시 붙어줘야합니다.